tests/test-stbt-py.sh

```diff
@@ -614,3 +614,35 @@ test_that_two_frames_iterators_can_return_the_same_frames_as_each_other() {
 	EOF
     stbt run -vv test.py || fail "Incorrect frames() behaviour"
 }
+
+test_that_press_returns_a_pressresult() {
+    cat > test.py <<-EOF &&
+	import time
+	
+	before = time.time()
+	result = stbt.press("KEY_MENU")
+	after = time.time()
+
+	assert before < result.start_time < result.end_time < after
+	assert result.key == "KEY_MENU"
+	assert isinstance(result.frame_before, stbt.Frame)
+
+	before = time.time()
+	result = stbt.press("KEY_HOME", hold_secs=3)
```

Comment:
This will add 3 seconds to the tests. It would be 1 second, or even shorter? You could also check that `result.end_time - result.start_time > $hold_secs`.

Reply:
Fixed.